contrib/forms/handwritten-notes/hw-notes.php

```diff
@@ -10,7 +10,7 @@
  * @license   https://github.com/openemr/openemr/blob/master/LICENSE GNU General Public License 3
  **/
 
-require_once("../../globals.php");
+require_once("../../../interface/globals.php");
```

Comment:
Would revert this since the path will be relative from interface/forms/handwritten-notes, and don't want it working from where it currently is.

Otherwise this PR is ready to come in :)

Reply:
I updated the PR to remove the fixes. I'll open separate PRs for the fixes, and for any ignore/configuration we want to do.